TestLockAndUnlock fails on semaphoreci

--- FAIL: TestLockAndUnlock (0.05s)
	lock_test.go:94: unexpected blocking
2015-12-14 08:01:42.941093 I | fileutil: purged file /tmp/purgefile800663965/0.test successfully
2015-12-14 08:01:42.941236 I | fileutil: purged file /tmp/purgefile800663965/1.test successfully
2015-12-14 08:01:42.941296 I | fileutil: purged file /tmp/purgefile800663965/2.test successfully
2015-12-14 08:01:42.951496 I | fileutil: purged file /tmp/purgefile800663965/3.test successfully
2015-12-14 08:01:42.961632 I | fileutil: purged file /tmp/purgefile800663965/4.test successfully
2015-12-14 08:01:42.971776 I | fileutil: purged file /tmp/purgefile800663965/5.test successfully
2015-12-14 08:01:42.982599 I | fileutil: purged file /tmp/purgefile800663965/6.test successfully
2015-12-14 08:01:42.993383 I | fileutil: purged file /tmp/purgefile351798872/0.test successfully
2015-12-14 08:01:42.993451 I | fileutil: purged file /tmp/purgefile351798872/1.test successfully
2015-12-14 08:01:42.993509 I | fileutil: purged file /tmp/purgefile351798872/2.test successfully
2015-12-14 08:01:42.993561 I | fileutil: purged file /tmp/purgefile351798872/3.test successfully
2015-12-14 08:01:42.993616 I | fileutil: purged file /tmp/purgefile351798872/4.test successfully
2015-12-14 08:01:43.005233 I | fileutil: purged file /tmp/purgefile351798872/5.test successfully
2015-12-14 08:01:43.005306 I | fileutil: purged file /tmp/purgefile351798872/6.test successfully
2015-12-14 08:01:43.027785 I | fileutil: purged file /tmp/purgefile969440485/0.test successfully
2015-12-14 08:01:43.027861 I | fileutil: purged file /tmp/purgefile969440485/1.test successfully
2015-12-14 08:01:43.027915 I | fileutil: purged file /tmp/purgefile969440485/2.test successfully
2015-12-14 08:01:43.038413 I | fileutil: purged file /tmp/purgefile969440485/3.test successfully
2015-12-14 08:01:43.048973 I | fileutil: purged file /tmp/purgefile969440485/4.test successfully
2015-12-14 08:01:43.058143 I | fileutil: purged file /tmp/purgefile969440485/5.test successfully
2015-12-14 08:01:43.068307 I | fileutil: purged file /tmp/purgefile969440485/6.test successfully
2015-12-14 08:01:43.080122 I | fileutil: purged file /tmp/purgefile277920512/0.test successfully
2015-12-14 08:01:43.080196 I | fileutil: purged file /tmp/purgefile277920512/1.test successfully
2015-12-14 08:01:43.080254 I | fileutil: purged file /tmp/purgefile277920512/2.test successfully
2015-12-14 08:01:43.080307 I | fileutil: purged file /tmp/purgefile277920512/3.test successfully
2015-12-14 08:01:43.080354 I | fileutil: purged file /tmp/purgefile277920512/4.test successfully
2015-12-14 08:01:43.091857 I | fileutil: purged file /tmp/purgefile277920512/5.test successfully
2015-12-14 08:01:43.091935 I | fileutil: purged file /tmp/purgefile277920512/6.test successfully
2015-12-14 08:01:43.114521 I | fileutil: purged file /tmp/purgefile585433901/0.test successfully
2015-12-14 08:01:43.114591 I | fileutil: purged file /tmp/purgefile585433901/1.test successfully
2015-12-14 08:01:43.114650 I | fileutil: purged file /tmp/purgefile585433901/2.test successfully
2015-12-14 08:01:43.124751 I | fileutil: purged file /tmp/purgefile585433901/3.test successfully
2015-12-14 08:01:43.134898 I | fileutil: purged file /tmp/purgefile585433901/4.test successfully
2015-12-14 08:01:43.145074 I | fileutil: purged file /tmp/purgefile585433901/5.test successfully
2015-12-14 08:01:43.155403 I | fileutil: purged file /tmp/purgefile585433901/6.test successfully
2015-12-14 08:01:43.167076 I | fileutil: purged file /tmp/purgefile802079912/0.test successfully
2015-12-14 08:01:43.167164 I | fileutil: purged file /tmp/purgefile802079912/1.test successfully
2015-12-14 08:01:43.167228 I | fileutil: purged file /tmp/purgefile802079912/2.test successfully
2015-12-14 08:01:43.167289 I | fileutil: purged file /tmp/purgefile802079912/3.test successfully
2015-12-14 08:01:43.167352 I | fileutil: purged file /tmp/purgefile802079912/4.test successfully
2015-12-14 08:01:43.179004 I | fileutil: purged file /tmp/purgefile802079912/5.test successfully
2015-12-14 08:01:43.179085 I | fileutil: purged file /tmp/purgefile802079912/6.test successfully

--- a/pkg/fileutil/lock_test.go
+++ b/pkg/fileutil/lock_test.go
@@ -78,7 +78,7 @@ func TestLockAndUnlock(t *testing.T) {
 	select {
 	case <-locked:
 		t.Error("unexpected unblocking")
-	case <-time.After(10 * time.Millisecond):
+	case <-time.After(100 * time.Millisecond):
 	}
 
 	// unlock
@@ -90,7 +90,7 @@ func TestLockAndUnlock(t *testing.T) {
 	// the previously blocked routine should be unblocked
 	select {
 	case <-locked:
-	case <-time.After(20 * time.Millisecond):
+	case <-time.After(1 * time.Second):
 		t.Error("unexpected blocking")
 	}
 }

